src/Menus.cpp

```diff
@@ -1320,18 +1314,12 @@ void showDisplayMenu(char* menuTitle) {
 }
 
 void saveSettings() {
-  bool stat = false;
-  if(writeConfig()) {
-    if(writeTmcConfig()){
-      #if defined(MULTISERVO)
-      if(writeServoMapping()) {
-        stat = true;
-      }
-      #else
-        stat = true;
-      #endif
-    }
-  }
+  bool stat = writeConfig()
```

Comment:
Please undo this change.
This is on purpose. If writeConfig returns  false, this means something is wrong with the SD-Card or memory and it doesn't make sense moving on.